### PR TITLE
Revamp README

### PR DESCRIPTION
**Pull request by yakkomajuri**
Copy edits and minor updates to the README.

Quite a lot of this is debatable, so feel free to push back!

### CODE REVIEW

File: charts/posthog/README.md [hunk 1]
```diff
@@ -90,8 +92,13 @@ Dependent charts can also have values overwritten. Preface values with postgresq
 
 All configuration options can be found in [ALL_VALUES.md](ALL_VALUES.md) or in [values.yaml](values.yaml) file.
 
-### Example `values.yaml`
-
+<details>
```
Comment (macobo) [inline charts/posthog/README.md:95]: Interesting trick! On the PR review screen the triangle to collapse/uncollapse is in the wrong place though

File: charts/posthog/README.md [hunk 2]
```diff
@@ -1,36 +1,38 @@
 # PostHog Helm Chart
 
-> :warning: **This chart is not ready yet** Be careful!
+### :warning: This chart is still under development! Proceed with caution.
```
Comment (macobo) [inline charts/posthog/README.md:3]: Nit: I like the previous style more but don't have a constructive reason why :)

Comment (yakkomajuri) [reply charts/posthog/README.md:3]: Completely fine, this is really subjective

File: charts/posthog/README.md (hunk 1 above)
Comment (yakkomajuri) [reply charts/posthog/README.md:95]: hmm - on my preview that was working fine

File: charts/posthog/README.md (hunk 2 above)
Comment (yakkomajuri) [reply charts/posthog/README.md:3]: hence the inline styling

File: charts/posthog/README.md (hunk 1 above)
Comment (yakkomajuri) [reply charts/posthog/README.md:95]: hence the inline styling

Comment (yakkomajuri) [reply charts/posthog/README.md:95]: worth seeing what it ends up looking like live

**Comment by macobo** [comment]
LGTM but will let tiina take a pass (will be on vacation next week, she's taking over the documentation effort)

**Comment by tiina303** [comment]
Thanks :) I'll put up some PRs later to update

**Review by tiina303 — APPROVED** [review]

**Comment by yakkomajuri** [comment]
hmm pretty weird that my preview showed inline correctly but that's not actually what happened

**Comment by tiina303** [comment]
> hmm pretty weird that my preview showed inline correctly but that's not actually what happened

what do you mean by this? Everything looked fine to me from a quick glance.

**Comment by yakkomajuri** [comment]
Was just a minor issue with the `details` tag. Fixed now.

https://github.com/PostHog/charts-clickhouse/pull/17#discussion_r654427002

**Pull request merged**